### ISSUE TITLE
[Translation tool] pl

### ISSUE THREAD

**Issue by maciek-szn** [briannesbitt/Carbon#2520]
## Additional fixes
I'm sorry, but my previous issue description (#2480) wasn't complete. Your changes in #2492 are insufficient.

For example you have corrected `za 1 godzina` to `za 1 godzinę` for `1 hour from now`. However if it's a part of a longer string, it's still incorrect.

Example: `1 hour and 30 minutes from now` is `za 1 godzina i 30 minut`, but it should be `za 1 godzinę i 30 minut`.

Some other examples:

- `a day and an hour from now` is `za dzień i godzina`, but it should be `za dzień i godzinę`,
- `1 minute 2 seconds from now` is `za 1 minuta 2 sekundy`, but it should be `za 1 minutę 2 sekundy`.

**Comment by kylekatarnls** [comment]
Hello, can you try `composer require nesbot/carbon:fix/pl-translations`?

**Comment by maciek-szn** [comment]
I'm trying, but I'm getting an error. However the fix looks good to me. And it's much cleaner than the previous one.

```
> composer require nesbot/carbon:dev-fix/pl-translations
./composer.json has been updated
Running composer update nesbot/carbon
Loading composer repositories with package information
Updating dependencies
Lock file operations: 0 installs, 1 update, 0 removals
  - Upgrading nesbot/carbon (2.55.2 => dev-fix/pl-translations ab33912)
Writing lock file
Installing dependencies from lock file (including require-dev)
Package operations: 0 installs, 1 update, 0 removals
  - Syncing nesbot/carbon (dev-fix/pl-translations ab33912) into cache
  - Removing nesbot/carbon (2.55.2)
    ab339127309cada3dc0ca6eaac29ef03ff536100 is gone (history was rewritten?)
    Update of nesbot/carbon failed

Installation failed, reverting ./composer.json and ./composer.lock to their original content.


  [RuntimeException]
  Failed to execute git checkout "ab339127309cada3dc0ca6eaac29ef03ff536100" -- && git reset --hard "ab339127309cada3dc0ca6eaac29ef03ff536100" --  

  fatal: reference is not a tree: ab339127309cada3dc0ca6eaac29ef03ff536100
```

**Comment by kylekatarnls** [comment]
Ah you have as version constraint via an other dependency, you can add this in your composer.json (or modify the nesbot/carbon line if there is one:

```json
"nesbot/carbon": "fix/pl-translations as 2.56.0",
```
And run `composer update nesbot/carbon`

**Comment by maciek-szn** [comment]
Something like that was necessary in composer.json:

`"nesbot/carbon": "dev-fix/pl-translations#78f98be83ae5142437c4949d9c9fd18fdb954b2b"`

I've done some testing and you can mark it as fixed. Thank you!

**Comment by kylekatarnls** [comment]
Thanks. It will be released in 2.56.0.
